iroha/src/block_sync.rs

```diff
@@ -63,7 +115,7 @@ impl BlockSynchronizer {
         let gossip_period = self.gossip_period;
```

Comment:
We should move contents of start function into `on_start` if block sync is an actor now